resources/views/auth/register.blade.php

```diff
@@ -47,6 +47,76 @@
                         </div>
 
                         <div class="form-group">
+                            <label for="firstname" class="col-md-4 control-label">First name</label>
+
+                            <div class="col-md-6">
+                                <input id="firstname" type="text" class="form-control" name="firstname" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="middlename" class="col-md-4 control-label">Middle name</label>
+
+                            <div class="col-md-6">
+                                <input id="middlename" type="text" class="form-control" name="middlename" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="lastname" class="col-md-4 control-label">Last name</label>
+
+                            <div class="col-md-6">
+                                <input id="lastname" type="text" class="form-control" name="lastname" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+                            <label for="email" class="col-md-4 control-label">Email address</label>
+
+                            <div class="col-md-6">
+                                <input id="email" type="text" class="form-control" name="email" required>
+
+                                @if ($errors->has('email'))
+                                    <span class="help-block">
+                                        <strong>{{ $errors->first('email') }}</strong>
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="telnum" class="col-md-4 control-label">Telephone number</label>
+
+                            <div class="col-md-6">
+                                <input id="telnum" type="text" class="form-control" name="telnum" required>
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <label for="telloc" class="col-md-4 control-label">Telephone local number</label>
+
+                            <div class="col-md-6">
+                                <input id="telloc" type="text" class="form-control" name="telloc" required>
+                            </div>
+                        </div>
```

Comment:
I guess this is unnecessary. Much better if we just remove the field "Telephone local number".